thirdeye/thirdeye-frontend/app/pods/home/share-dashboard/template.hbs

```diff
@@ -11,24 +11,53 @@
     <article class="share-dashboard-container__share-configure">
       <h4 style="color: rgba(0,0,0,0.9);font-size: 18px;font-weight: bold;">Configure summary</h4>
       <section>
-        <div style="margin-left: 15px;margin-bottom: 15px;display: inline-block;">
-          <p><input type="checkbox" {{action 'toggleSelection' "dashboard_summary" on="change"}} checked> Add dashboard-level comments</p>
-          <p>
-            <textarea {{action 'updateComment' "dashboard_summary" on="change"}} value="{{dashboard_summary_comment}}" placeholder="Comment for dashboard summary" name="dashboard_summary" id="dashboard_summary" style="border: none;border-bottom: 1px solid grey;margin-left: 15px;width: 800px;">
-            </textarea>
-          </p>
+        <div style="margin-left: 15px;display: inline-block;">
+          <p><input type="checkbox" {{action 'toggleSelection' "dashboard_summary" on="change"}}> Add dashboard-level comments</p>
+          {{#if showDashboardSummary}}
+            <p>
+              <textarea {{action 'updateComment' "dashboard_summary" on="change"}} value="{{dashboard_summary_comment}}" placeholder="Comment for dashboard summary" name="dashboard_summary" id="dashboard_summary" style="border: none;border-bottom: 1px solid grey;margin-left: 15px;width: 800px;">
+              </textarea>
+            </p>
+          {{/if}}
         </div>
-        {{#x-tree
-          chosenId=selectedNode
-          checkable=true
-          expandDepth=0
-          model=tree as |node|}}
-            {{node.name}}
-            {{#if node.isComment}}
-              <input type="text" {{action 'updateComment' node.id on="change"}} value="{{node.comment}}" placeholder="Add comments for metrics/alerts" name="{{node.name}}" id="{{node.id}}" style="border: none;border-bottom: 1px solid grey;margin-left: 30px;width: 800px;">
+      </section>
+      <section>
+        <p>
+          {{#x-tree
```

Comment:
Curious about the UI for this tree component - maybe throw in a screenshot if possible.